latex/accessors.tex

```diff
@@ -29,9 +29,9 @@
 
 Because of this the interface of the \codeinline{accessor} will
 be different depending on the possible combinations of those parameters. There
-are three main categories of accessor; buffer accesors (see Section
-~\ref{sub.section.accessors.buffer}), local accessors (see Section
-~\ref{sub.section.accessors.local}) and image accessors (see Section
+are three main categories of accessor; buffer accesors (see Section%
```

Comment:
Are these % intentional?  I can't figure out what the impact is.

Reply:
Yes. It was a bug in the specification I fixed while I was reading it.
There was a potential double space because of the new-line (which is a space) + "~" which is already a non-breakable space...
So adding a % at the end 
Classical LaTeX trick, explaining why there are often a lot of % at some end of line when spaces matter.